reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -162,6 +164,17 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
         }
     }
 
+    private fun createScopes() {
+        input.ortResult.getProjects().flatMap {
+                project -> project.scopes.map { it.name }
```

Comment:
Wrong formatting: `project ->` belongs to the previous line.

Reply:
Done.